packages/core/src/components/Attachments.tsx

```diff
@@ -22,6 +22,7 @@ const renderChildNodes = createRenderChildNodes({
   section: withSectionContext(RenderChildNodes)
 })
 
+// eslint-disable-next-line no-empty-pattern
```

Comment:
Mieluummin esim. `_: ExamProps`.

Reply:
Silloin tulee ESLint: '_' is defined but never used.(@typescript-eslint/no-unused-vars)

Onko se parempi?

Reply:
Onko _ muutenkaan hyvä? Sitä käytetään lodash:lle. Noissa funkkareissa ei vielä mut jos joku sattuu lisäämään sen käytön niin tulee hämäriä virheitä

Reply:
Meidän pitäisi lisätä linttikonffiin `{ "argsIgnorePattern": "^_" }`. Se vastaa täsäkääntäjän konventiota. Myös esim `_props` kelvannee.

Reply:
Mä vaihdan _props:ksi nämä. Tehdään lintterikonffiksen päivityksen erikseen myöhemmin.